### ISSUE TITLE
Notification for earning a new badge

### ISSUE THREAD

**Issue by jhenshall** [mapswipe/mapswipe#134]
I just happened to notice on the account page that I passed the threshold for a new badge but I was never alerted to this happening.

Can we have a full page notification rewarding the user for earning a new badge. I think this happened on the old version but not certain?

**Comment by laurentS** [comment]
It looks like you're right indeed. I had completely forgotten about that bit, which I probably threw out when I first rewrote stuff. Can we tag it for release `2.0.2`?

**Comment by jhenshall** [comment]
Of course! Tag for whenever you think viable. Just wanted to add to backlog. There's also ticket #131  around how we display badges and what they mean to users, which could be part of the new designs.
